src/App.js

```diff
@@ -1,43 +1,20 @@
 import 'src/assets/styles/_global.scss';
 
 import CssBaseline from '@mui/material/CssBaseline';
-import React, { useEffect, useState } from 'react';
-import { Route, Routes, useLocation } from 'react-router-dom';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Header from 'src/components/Header';
-import Home from 'src/pages/home';
-
-import LinearIndeterminate from './components/Loader';
-import Careers from './pages/careers';
+import Careers from 'src/pages/careers';
+import routes from 'src/router/routes';
 
 function App() {
-  const [progress, setProgress] = useState(true);
-  const [prevLoc, setPrevLoc] = useState('');
-  const location = useLocation();
-
-  useEffect(() => {
-    setPrevLoc(location.pathname);
-    setProgress(true);
-    if (location.pathname === prevLoc) {
-      setPrevLoc('');
-    }
-  }, [location]);
-
-  useEffect(() => {
-    setTimeout(() => {
-      setProgress(false);
-    }, 700);
-  }, [prevLoc]);
-
   return (
     <>
       <CssBaseline />
       <Header />
-      {progress ? <LinearIndeterminate /> : (
-        <Routes>
-          <Route exact path="/" element={<Home tabTitle="Fakng - Home page" />} />
-          <Route path="careers" element={<Careers tabTitle="Fakng - Careers page" />} />
-        </Routes>
-      )}
+      <Routes>
+        <Route path={routes.careers} element={<Careers tabTitle="Fakng - Careers page" />} />
```

Comment:
Can we have tabTitle just "Careers page"?

Reply:
Yep 👍